backend/src/Repository/CaptainFinancialDuesEntityRepository.php

```diff
@@ -164,4 +164,25 @@ public function getFinancialDuesByCaptainId(int $captainId): array
             ->getQuery()
             ->getResult();
     }
+
+    public function getLatestCaptainFinancialDuesByUserId(int $userId): CaptainFinancialDuesEntity
```

Comment:
As the return query result is one or null, then the function may return NULL in return.